OWNERS_ALIASES

```diff
@@ -7,11 +7,16 @@ aliases:
     - andreyvelich
     - gaocegege
     - johnugeorge
+  wg-notebooks-leads:
+    - StefanoFioravanzo
+    - elikatsis
+    - kimwnasptd
+    - thesuperzapper
   wg-pipelines-leads:
     - Bobgy
     - animeshsingh
     - james-jwu
-    - kramachandran
+    - karkumar
```

Comment:
@jlewi after running the `make generate` [ with the fix from #417 ] this was also changed.

Should I revert this changed line? If yes how can we assure that other people won't bump into this in the future?

Reply:
Looks like the bug is here:
https://github.com/kubeflow/community/blob/abb090d9af844354ee1c71b93c58f713b3961ece/wgs.yaml#L116

That should be changed to @kramachandran since that is the GitHub id that he wants to use; looks like OWNER_ALIASES is autogenerated from that.

Reply:
ACK, I'll push another commit that changes @kramachandran handle in the `wgs.yaml`

Reply:
alright, the bot removed the `do-not-merge/invalid-owners-file` label